<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Sigmet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="headingGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Heading</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="0" column="2">
       <widget class="QLabel" name="sequenceLabel">
        <property name="text">
         <string>Sequence</string>
        </property>
        <property name="buddy">
         <cstring>sequence</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="endingTimeLabel">
        <property name="text">
         <string>Ending</string>
        </property>
        <property name="buddy">
         <cstring>endingTime</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="sequence"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="beginningTime"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="beginningTimeLabel">
        <property name="text">
         <string>Beginning</string>
        </property>
        <property name="buddy">
         <cstring>beginningTime</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="endingTime"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="typeGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="phenomenaLabel">
        <property name="text">
         <string>Phenomena</string>
        </property>
        <property name="buddy">
         <cstring>phenomena</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Description</string>
        </property>
        <property name="buddy">
         <cstring>description</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="description"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="phenomena"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="obsFcstGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Observation / Forecast</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="observationTimeLabel">
        <property name="text">
         <string>Observation Time</string>
        </property>
        <property name="buddy">
         <cstring>observationTime</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="observationLabel">
        <property name="text">
         <string>Observation</string>
        </property>
        <property name="buddy">
         <cstring>observation</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="observationTime"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="observation"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="forecastTimeLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Forecast Time</string>
        </property>
        <property name="buddy">
         <cstring>forecastTime</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="forecastTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="extentGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Extent</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <widget class="QLabel" name="levelLabel">
        <property name="text">
         <string>Fight Level</string>
        </property>
        <property name="buddy">
         <cstring>level</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="baseLabel">
        <property name="text">
         <string>Base</string>
        </property>
        <property name="buddy">
         <cstring>base</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="base"/>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="top"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="level"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="topLabel">
        <property name="text">
         <string>Top</string>
        </property>
        <property name="buddy">
         <cstring>top</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="changeGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Change</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QLabel" name="speedLabel">
        <property name="text">
         <string>Speed</string>
        </property>
        <property name="buddy">
         <cstring>speed</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="speed"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="direction">
        <item>
         <property name="text">
          <string notr="true">N</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">NNE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">NE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">ENE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">E</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">ESE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">SE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">SSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">SSW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">SW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">WSW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">W</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">WNW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">NW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">NNW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">STNR</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="directionLabel">
        <property name="text">
         <string>Direction</string>
        </property>
        <property name="buddy">
         <cstring>direction</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="intensityChangeLabel">
        <property name="text">
         <string>Intensity</string>
        </property>
        <property name="buddy">
         <cstring>intensityChange</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="intensityChange">
        <item>
         <property name="text">
          <string notr="true">NC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">INTSF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">WKN</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>beginningTime</tabstop>
  <tabstop>endingTime</tabstop>
  <tabstop>sequence</tabstop>
  <tabstop>description</tabstop>
  <tabstop>phenomena</tabstop>
  <tabstop>observation</tabstop>
  <tabstop>observationTime</tabstop>
  <tabstop>forecastTime</tabstop>
  <tabstop>level</tabstop>
  <tabstop>base</tabstop>
  <tabstop>top</tabstop>
  <tabstop>direction</tabstop>
  <tabstop>speed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
